### PR TITLE
Add IAM Assume Role support

### PR DESCRIPTION
**Pull request by ipcrm**

### CODE REVIEW

File: lib/EcsSupport.ts [hunk 1]
```diff
@@ -32,8 +31,11 @@ export function ecsSupport(): ExtensionPack {
             "sdm.aws.accessKey",
             "sdm.aws.secretKey",
             {path: "sdm.aws.ecs.desiredCount", type: ConfigurationValueType.Number},
-            // {path: "sdm.aws.ecs.networkConfiguration", type: ConfigurationValueType.Object}, -> Add this once supported
-            // ^^https://github.com/atomist/sdm/issues/580
+
+            /** -> Add these once supported (https://github.com/atomist/sdm/issues/580)
```
Comment (cdupuis) [inline lib/EcsSupport.ts:35]: Thanks for the reminder!

File: lib/goals/EcsDeploy.ts [hunk 2]
```diff
@@ -105,19 +108,20 @@ export interface EcsDeployment extends Deployment {
...
+export class EcsDeployer implements Deployer<EcsDeploymentInfo & EcsDeployRegistration, EcsDeployment> {
     constructor(private readonly projectLoader: ProjectLoader) {
     }
 
     public async deploy(da: DeployableArtifact,
                         esi: EcsDeploymentInfo,
                         log: ProgressLog,
                         credentials: ProjectOperationCredentials): Promise<EcsDeployment[]> {
-        logger.info("Deploying app [%j] to ECS [%s]", da, esi.description);
+        log.write(`Deploying service [${da.name}] to ECS [${esi.cluster}]`);
```
Comment (cdupuis) [inline lib/goals/EcsDeploy.ts]: We don't use [] in progressLog messages. I'd change that to `Deploying service '${da.name}' to ECS '${esi.cluster}'?

**Review by cdupuis — APPROVED** [review]
Minor comment. But looks good.

File: lib/goals/EcsDeploy.ts (hunk 2 above)
Comment (ipcrm) [reply lib/goals/EcsDeploy.ts]: Ah, np.  Updating.

**Pull request merged**